### PR TITLE
DocuBlocks for shard distribution endpoints

### PR DESCRIPTION
**Pull request by Simran-B**
Add missing docs for `/_admin/cluster/shardDistribution` and `/_api/database/shardDistribution` added in #13505.

Unclear whether `/_admin/cluster/collectionShardDistribution` should be documented as well (missing in CHANGELOG).

Needs backports: 3.8

### CODE REVIEW

File: Documentation/DocuBlocks/Rest/Cluster/get_cluster_shard_distribution.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+@startDocuBlock get_cluster_shard_distribution
+@brief allows to query the global shard distribution statistics in the cluster
```
Comment (jsteemann) [inline Documentation/DocuBlocks/Rest/Cluster/get_cluster_shard_distribution.md:3]:
```suggestion
@brief provides the global shard distribution statistics in the cluster
```

File: Documentation/DocuBlocks/Rest/Database/get_api_database_shard_distribution.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+
+@startDocuBlock get_api_database_shard_distribution
+@brief allows to query the shard distribution statistics of the current database in the cluster
```
Comment (jsteemann) [inline Documentation/DocuBlocks/Rest/Database/get_api_database_shard_distribution.md:3]:
```suggestion
@brief provides the shard distribution statistics of the current database in the cluster
```

**Review by jsteemann — APPROVED** [review]

**Comment by Simran-B** [comment]
Example generation currently broken: https://arangodb.atlassian.net/browse/DOC-94

**Comment by Simran-B** [comment]
The current policy appears to be to not document internal APIs at all.

**Pull request closed without merging**